### PR TITLE
some edits, not all

### PR DESCRIPTION
**Pull request by BeckySharp**
Added some patterns, changed some examples.  More edits coming, but I wanted these changes to serve as a basis for those edits, which will address the issues posted

FYI @zupon -- please use this branch as the basis (unless if gets merged, then use master)

### CODE REVIEW

File: CompositionalOntology_v2.1_metadata.yml [hunk 1]
```diff
@@ -113,7 +113,9 @@
               name: hopper_band
               polarity: -1
             - OntologyNode:
-              patterns:
+              pattern:
+              - locust
+              - \bDL\b
```
Comment (MihaiSurdeanu) [inline CompositionalOntology_v2.1_metadata.yml:118]: @BeckySharp : what is this?

**Review by MihaiSurdeanu — APPROVED** [review]
Looks good!

**Comment by BeckySharp** [comment]
An abbreviation for “desert locust” I think

**Comment by BeckySharp** [comment]
@chanys I'm going to merge this bc it's primarily a change to the metadata, but we can roll it back if you don't want it, thanks!

**Pull request merged**